### PR TITLE
Add Jriyya Personal Page 

### PR DESCRIPTION
**Pull request by jriyyya**
## Description

I have added my personal page to the builders route

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Screenshot
image

Your ENS/address: `0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155`

### CODE REVIEW

File: packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import me from "./me.webp";
+import "./style.css";
+
+export default function Jriyyya() {
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx:8]: Nitpick: 

For pages, In SE-2 we tend to use arrow functions for page and then export it as default at bottom, checkout `app/page.tsx` for example. 


Just trying to stay consistent. but its completely fine if we use function and default export directly !

**Review by technophile-04 — CHANGES_REQUESTED** [review]
Looks cool !! Thanks @jriyyya ! Just added a couple of comments could you please fix them ?

File: packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      className="absolute text-purple-400 dark:text-purple-300"
+      key={duration}
+      style={{
+        animation: `anim--star ${duration}ms`,
+        top: `${position.y}%`,
+        left: `${position.x}%`,
+        fontSize: `${((duration - durationStep) / durationStep) * 64}px`,
+      }}
+    >
+      {Math.random() < 0.5 ? "+" : <>&#9733;</>}
```
[the remark this answers is not in the document]
Comment (jriyyya) [reply packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx:102]: I feel like just the stars would nice, so removing the + 😁

File: packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx (hunk 1 above)
Comment (jriyyya) [reply packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx:8]: Done

**Comment by jriyyya** [comment]
@technophile-04 Thanks for the review, I have made the suggested changes

**Comment by technophile-04** [comment]
Thanks @jriyyya ! One last small change could just maybe make it a bit more responsive on small screens, maybe use `flex-col` for small devices ? because currently this is how it looks on small screens : 
Screenshot 2024-06-06 at 5 12 11 PM

**Comment by derrekcoleman** [comment]
Hi @jriyyya! Just had to hop in and say how slick your page looks 🔥🔥🔥

**Comment by jriyyya** [comment]
@technophile-04 I completely forgot about it, Fixed it
image

**Comment by jriyyya** [comment]
@derrekcoleman Thankk you! 😁

**Comment by technophile-04** [comment]
@jriyyya did you forgot to push the changes ?

**Comment by jriyyya** [comment]
@technophile-04 Faulty network xd

**Review by technophile-04 — APPROVED** [review]
Thanks !! Looks awesome !

**Pull request merged**